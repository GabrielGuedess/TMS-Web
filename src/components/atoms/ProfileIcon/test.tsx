import { render } from '@testing-library/react';

import { ProfileIcon } from '.';

describe('<ProfileIcon />', () => {
  it('should render the profile icon correctly', () => {
    const { container } = render(<ProfileIcon />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
