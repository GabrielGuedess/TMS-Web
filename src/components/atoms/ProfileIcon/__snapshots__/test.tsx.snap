// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfileIcon /> should render the profile icon correctly 1`] = `
<svg
  fill="none"
  height="24"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
>
  <path
    d="M20 21v-2a4 4 0 0 0-4-4H8a4 4 0 0 0-4 4v2"
  />
  <circle
    cx="12"
    cy="7"
    r="4"
  />
</svg>
`;
